spec/models/form526_submission_spec.rb

```diff
@@ -23,6 +23,40 @@
     File.read('spec/support/disability_compensation_form/submissions/only_526.json')
   end
 
+  describe 'submit_endpoint enum' do
+    context 'when submit_endpoint is evss' do
+      let(:submit_endpoint) { 'evss' }
```

Comment:
Are these assignments actually setting anything? I don't see `submit_endpoint` set in the `subject` block above am I missing something?

Reply:
Good catch. Corrected.

Reply:
Could you include a link to the commit where the change was made when replying to a change request? I had a lead engineer who required that once and I thought it was a good system, makes it easy to check the change.